aviatrix/resource_aviatrix_gateway.go

```diff
@@ -1474,9 +1474,21 @@ func resourceAviatrixGatewayRead(d *schema.ResourceData, meta interface{}) error
 	}
 
 	if goaviatrix.IsCloudType(gw.CloudType, goaviatrix.AWSRelatedCloudTypes|goaviatrix.AzureArmRelatedCloudTypes) {
```

Comment:
Why is fetching tags gated on these particular cloud types? IIUC google supports tags too. Suggest we always record the flags if we get them.